sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -416,18 +415,18 @@ def upload_blob_from_url(self, source_url, **kwargs):
             source_url=source_url,
             **kwargs)
         try:
-            return self._client.block_blob.put_blob_from_url(**options)
+            return self._client.block_blob.put_blob_from_url(**options)  # type: ignore
```

Comment:
I believe the pattern for returns from generated code using a cls was just using `cast`.